xbmc/guilib/GUIWindow.cpp

```diff
@@ -792,8 +791,7 @@ void CGUIWindow::FreeResources(bool forceUnload /*= false */)
   if (m_loadType == LOAD_EVERY_TIME || forceUnload) ClearAll();
   if (forceUnload)
   {
-    delete m_windowXMLRootElement;
-    m_windowXMLRootElement = nullptr;
+    m_windowXMLRootElement.release();
```

Comment:
Better use `reset()`, which also does what you want but returns `void`, in difference to `release`which returns the raw pointer to the released object. But you do not need that pointer, right?